src/main/java/net/kodehawa/mantarobot/db/entities/helpers/PlayerData.java

```diff
@@ -72,7 +72,7 @@ public PlayerData() { }
     //LEGACY SUPPORT
     //Marriage UUID data is on UserData now!
     public boolean isMarried() {
-        return marriedWith != null && MantaroBot.getInstance().getShardManager().getUserById(marriedWith) != null;
+        return marriedWith != null && MantaroBot.getInstance().getShardManager().retrieveUserById(marriedWith).complete() != null;
```

Comment:
will throw if id doesn't exist (deleted account?)

Reply:
Removed this, as we don't actually use it anymore :tm: